point-of-care.md

```diff
@@ -65,5 +65,34 @@ requirements a bit, the code may look like this:
 If we do this, we need a font where the 0 and the O are really different, or
 maybe underline the digits.
 
+### Another alternative
 
+We can generate 16 random bytes and use this as random authentication code.
 
+We can format this output as 6 numbers of 6 digits in length.
+
+This gives us an field of 10^36 which translates to an entropy of 
+ln(10)/ln(2) which is about 119 bits which puts us in the realm of the
+complexity of 3DES.
+
+Scanning a QR-code is easiest, but the healthcare worker can also read out
+these numbers over the phone.
+
+So we would display:
+
+"qr code"
```

Comment:
I would also suggest to send the code by SMS which can be picked up by the mobile app. There is a very big chance that the phone which is used for calling with the medical specialist is the same as the one on which the app is installed. In that way it's quite hard to keep on the line with the medical specialist and open the app to enter the digits. Again, especially for the less digital aware persons around this would be quite a task.

Reply:
> I would also suggest to send the code by SMS which can be picked up by the mobile app. There is a very big chance that the phone which is used for calling with the medical specialist is the same as the one on which the app is installed. In that way it's quite hard to keep on the line with the medical specialist and open the app to enter the digits. Again, especially for the less digital aware persons around this would be quite a task.

This is a very good idea, but I want to note that "access to SMS-messages" should be given on a voluntary basis. Therefore, it should also be possible to copy the numbers from the sms-app with pen and paper, and then re-enter them into the contact-tracing app.

Reply:
Yes I agree that this feature should be on voluntary basis.

The calculation about the SMS however is incorrect. I am not in the SMS business but I expect that nowadays it can be done way more cheaper. Beside that, there will never be 17 million messages sent taking into account that there will not be a 100% app coverage for sure: people don't want to use the app, people who doesn't have a smart phone, kids etc.

Reply:
> The calculation about the SMS however is incorrect. I am not in the SMS business but I expect that nowadays it can be done way more cheaper.

So do I, but it's good to know that in the worst-case we can buy services from any regular sms-gateway and still be within reasonable limits.

Reply:
I would not scale to 100%, at least not at the start. Looking at countries that have implemented some sort of app and looking at trust in the Netherlands, this solution would be doing very good if it hits 60% adoption in the age 10+(~15 million*). This calculates to 9 million people. I would also consider this in [#10](https://github.com/ahupowerdns/covid-backend/issues/10)

* https://opendata.cbs.nl/statline/#/CBS/en/dataset/03743eng/table?ts=1586852525387

Reply:
A worst case scenario (everyone gets diagnosed infected & uses the app) which could generate a bill of € 1.7M for sending € 0.10 text messages is **not** a problem for the Dutch government to pay.

Reply:
> I would not scale to 100%, at least not at the start. Looking at countries that have implemented some sort of app and looking at trust in the Netherlands, this solution would be doing very good if it hits 60% adoption in the age 10+(~15 million*). This calculates to 9 million people. I would also consider this in [#10](https://github.com/ahupowerdns/covid-backend/issues/10)

Good point!
But if we can use a static file-format which we can put on a CDN, than we can even ignore the scaling issues entirely. The calculations in #10 are the way they are because this helps us getting an upper bound on the costs we might have to make. Whether that be in servers or in CDN-delivery costs.

> A worst case scenario (everyone gets diagnosed infected & uses the app) which could generate a bill of € 1.7M for sending € 0.10 text messages is **not** a problem for the Dutch government to pay.

This is also an estimation of an upper-bound. A quick search on Google returns multiple sms-gateways that can deliver 10k sms-messages for about 6,6 cents. The upper bound just became 1122000 euro's.

It's also worth noting that if privacy becomes an issue, I can build or script something with an old smartphone or an arduino with a GSM-shield that can push all these messages out without going through a 3rd party API.

But I expect the government already has something in place that we can use or that one of the major telecom-providers even has something much better (those API's must talk to something). Some might even want to donate their services to us for free, because this is relatively cheap PR for them. (We all know that an sms-message has a true cost-price of less than 0,1 cents for them, so it would not even be 17k euro's they are donating.)

Reply:
By the way! Thanks for all the feedback everyone!
You guys are a tremendous help en organizing thoughts and evaluating possibilities!